gtdynamics/dynamics/Chain.h

```diff
@@ -0,0 +1,84 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2021, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file Chain.h
+ * @brief Create a serial kinematic chain.
+ * @author Dan Barladeanu, Frank Dellaert.
+ */
+
+
+#pragma once
+
+//#include <gtsam/linear/NoiseModel.h>
```

Comment:
kill commented out stuff. Try to remove as many headers as you can.

Reply:
done